data/bactrian.txt

```diff
@@ -138,7 +138,7 @@ mission "Bactrian Unlock [0]: Haven"
 		has "Bactrian Unlock [0]: active"
 	on offer
 		conversation
-			`You spend a few hours scanning the surface of the planet in an attempt to find the Star Queen, but nothing comes up. It will likely require military-grade scanners in order to find the ship, which Lieutenant Paris will undoubtedly have.`
+			`You spend a few hours scanning the surface of the planet in an attempt to find the Star Queen, but nothing comes up. It will likely require military-grade scanners in order to find the ship, which the Deep Security will undoubtedly have.`
```

Comment:
"We're being overrun by pirates!! AHhh! Call Deep Security!"
and
"We're being overrun by pirates!! AHhh! Call **the** Deep Security _station/fleet/ship/officer_"
those are OK to me.
But " ... Call the Deep Security!" is not.

In this context, where "Deep Security" is an organizational label, no article. This is like: "Bob will call the Microsoft for tech support," vs "Bob will call Microsoft for tech support."